### PR TITLE
DMN 1.3 0093-feel-at-literals

### PR DESCRIPTION
**Pull request by StrayAlien**
assertion on the new @ literal syntax

### CODE REVIEW

**Comment by StrayAlien** [comment]
A few small additions to this PR to assert datetimes with zone and offset info, and same for time.

File: TestCases/compliance-level-3/0093-feel-at-literals/0093-feel-at-literals.dmn [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+             xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/"
+             xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/"
+             xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL @ literals</description>
+
+    <decision name="test_001" id="_test_001">
+        <variable name="test_001"/>
+        <literalExpression>
+            <text>@"foo" = null</text>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0093-feel-at-literals/0093-feel-at-literals.dmn:15]: My understanding is that you want to test that an At-literal must conform to being one of the temporal cases from the specification.

I'm pretty sure in analogous cases, we tested for returning null in the test case, with an helper flag.

I believe is meaningful to keep the same strategy to test errors.

Hence, I believe this test shall be amended in the FEEL expression to be, more simply:
```
<text>@"foo"</text>
```

and test case for expecting
```
<value xsi:nil="true"/>
```

**Review by tarilabs — COMMENTED** [review]
Thank you for this contributions!

I have 1 comment concerning the "error-related" test case.

File: TestCases/compliance-level-3/0093-feel-at-literals/0093-feel-at-literals.dmn (hunk 1 above)
Comment (StrayAlien) [reply TestCases/compliance-level-3/0093-feel-at-literals/0093-feel-at-literals.dmn]: Thanks @tarilabs . Fair call.  Amended.

**Comment by opatrascoiu** [comment]
Looks good to me. Thank you @StrayAlien

File: TestCases/compliance-level-3/0093-feel-at-literals/0093-feel-at-literals.dmn (hunk 1 above)
Comment (tarilabs) [reply TestCases/compliance-level-3/0093-feel-at-literals/0093-feel-at-literals.dmn]: this was actioned with 50e8432, marking https://github.com/dmn-tck/tck/pull/391#discussion_r584096252 as resolved.

**Review by tarilabs — APPROVED** [review]
https://github.com/dmn-tck/tck/pull/391#pullrequestreview-600123266 was acted,
approving, again thank you @StrayAlien !

**Pull request merged**